### ISSUE TITLE
Tag and Release Version 0.3.0?

### ISSUE THREAD

**Issue by timgws** [OpenSkill/Datatable#33]
@Chumper what do you say to me finishing up some documentation on the QBProvider in the `documentation` class, and then you removing the scary warning on the `master` `README.md`?

To me, it seems that `QBProvider`, `Datatable19QueryParser` and `Datatable19Version` has at least the core functionality to replace general usages from `Chumper/Datatable`.

The only things that would be blocking this (AFAIK) is:
- #26 - [CollectionProvider] Returned columns are empty
- #32 - Reduce complexity of search query parsing. Seperate query preparation

**Comment by Chumper** [comment]
Yes, seems reasonable, if you feel safe to use it instead of the old version then we can remove the warning :)
